installation-bundle/src/Database/Installer.php

```diff
@@ -255,6 +255,10 @@ private function checkEngineAndCollation(array &$sql, array &$order, Schema $fro
             if (strtolower($tableOptions->Engine) !== strtolower($engine)) {
                 if ($innodb && $dynamic) {
                     $command = 'ALTER TABLE '.$tableName.' ENGINE = '.$engine.' ROW_FORMAT = DYNAMIC';
+
+                    if (false !== stripos($tableOptions->Create_options, 'key_block_size=')) {
```

Comment:
Can we be sure that there is never a space between `key_block_size` and `=`?

Reply:
I think so, yes. In my tests there was never a space. But we could also check for `/key_block_size\s*=/i` instead to be sure.

Reply:
But we would have to adjust the `row_format=dynamic` check too then.

Reply:
I think so too, otherwise `row_format=dynamic` would fail as well. As we never had issues with that, I propose to keep it that way 🤷